windows/scripts/MatlabPostBuild.cmd

```diff
@@ -0,0 +1,12 @@
+set SOLUTION_DIR=%~1%
+set OUTPUT_DIR=%~2%
+
+echo MatlabPostBuild.cmd : copy matlab generated scripts to output.
+
+copy /y "%SOLUTION_DIR%..\matlab\+caffe\*.m" "%OUTPUT_DIR%matcaffe\+caffe"
```

Comment:
@mnidza @lunzueta If we need a clean release. We can change the prebuild and postbuild scripts to:
```batch
set SOLUTION_DIR=%~1%
set OUTPUT_DIR=%~2%

echo MatlabPreBuild.cmd : Create output directories for matlab scripts.

if not exist "%OUTPUT_DIR%\matcaffe" mkdir "%OUTPUT_DIR%\matcaffe"
if not exist "%OUTPUT_DIR%\matcaffe\+caffe" mkdir "%OUTPUT_DIR%\matcaffe\+caffe"
if not exist "%OUTPUT_DIR%\matcaffe\+caffe\private" mkdir "%OUTPUT_DIR%\matcaffe\+caffe\private"
```

```batch
set SOLUTION_DIR=%~1%
set OUTPUT_DIR=%~2%

echo MatlabPostBuild.cmd : copy matlab generated scripts to output.

@echo run_tests.m > "%temp%\excludelist.txt"
xcopy /y "%SOLUTION_DIR%..\matlab\+caffe\*.m" "%OUTPUT_DIR%matcaffe\+caffe" /exclude:%temp%\excludelist.txt
copy /y "%SOLUTION_DIR%..\matlab\+caffe\private\*.m" "%OUTPUT_DIR%matcaffe\+caffe\private"
move /y "%OUTPUT_DIR%caffe_.*" "%OUTPUT_DIR%matcaffe\+caffe\private"
```